daemons/attrd/attrd_ipc.c

```diff
@@ -269,7 +269,7 @@ expand_regexes(xmlNode *xml, const char *attr, const char *value, const char *re
                 /* Copy all the attributes from the parent over, but remove the
                  * regex and replace it with the name.
                  */
-                attrd_copy_xml_attributes(xml, child);
+                pcmk__xe_copy_attrs(child, xml, false, false);
```

Comment:
worth updating comment

Reply:
sounds good, will say "non-conflicting attributes", the rest still seems fine but let me know if you think of something else